extensions/email/src/main/java/org/mule/extension/email/api/EmailAttributes.java

```diff
@@ -93,28 +108,40 @@
      * <p>
      * If this is a sent email this will be null.
      */
-    private final LocalDateTime receivedDate;
+    @Parameter
+    @Optional
+    private LocalDateTime receivedDate;
 
     /**
      * The time where the email was sent.
      * <p>
      * Different {@link Folder} implementations
      * may assign this value or not.
      */
-    private final LocalDateTime sentDate;
+    @Parameter
+    @Optional
+    private LocalDateTime sentDate;
```

Comment:
that's something to be fixed (Actually, you will fix it yourself next sprint). That doesn't mean that the javadoc isn't the right place to do it